pkg/build/strategies/layered/layered.go

```diff
@@ -70,6 +74,9 @@ func getDestination(config *api.Config) string {
 // accessible and non-empty directory.
 func checkValidDirWithContents(name string) bool {
 	items, err := ioutil.ReadDir(name)
+	// NOTE: obfuscated and unhandled error
+	// what if we don't have read rights to the repo?
+	// this should be replaced with a generic print.
```

Comment:
that's the callers problem to decide how to deal with...  even this warning probably doesn't belong in this function.